src/modules/auth/interfaces/jwt-payload.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface JwtPayload {
+  userId: number;
+  uuid: string;
+  email: string;
+  isArtist: boolean;
```

Comment:
Isso fica fechado pra alterações, se futuramente for adicionado outro tipo de usuário?

Reply:
A principal vantagem do Open-closed é a flexibilidade das funcionalidades no sistema.
Esse principio não tem valor em casos em que não se faz uso em mais de uma dependencia, criando assim, uma poluição e aumento da complexidade sem necessidade.